src/css/custom.css

```diff
@@ -62,12 +64,16 @@ html[data-theme='dark'] .navbar-item-github:before {
   background-position: center;
 }
 
-@media screen and (max-width: 996px) {
+@media screen and (max-width: 1024px) {
```

Comment:
With the added `Tournaments` label I opted to hide the social icons a touch earlier

This fixes an issue where we ran out of horizontal space JUST before the mobile breakpoint

Doesn't seem like its easy to _change_ that breakpoint to opted for this!